experimental/SetPartitions/src/SetPartitions.jl

```diff
@@ -51,6 +63,10 @@ export set_partition
 export spatial_partition
 export upper_colors
 export upper_points
+export simplify_operation_zero
+export simplify_operation
```

Comment:
```suggestion
```
in my opinion, these should not get exported, as they are just used to convert user input to something consistent in the constructor. So a user should never need to call these by themselves